mmflow/models/flow_estimators/flownet.py

```diff
@@ -18,75 +18,71 @@ def __init__(self, *args, **kwargs) -> None:
         super().__init__(*args, **kwargs)
 
     def extract_feat(self, imgs: Tensor) -> TensorDict:
-        """Extract feature.
+        """Extract features from images.
 
         Args:
-            imgs (Tensor): Input images.
+            imgs (Tensor): The concatenated input images.
 
         Returns:
-            TensorDict: The dict of tensor.
+            TensorDict: The feature pyramid extracted from the concatenated
+                input images.
         """
         return self.encoder(imgs)
 
-    def loss(self, batch_inputs: Tensor,
-             batch_data_samples: SampleList) -> dict:
+    def loss(self, inputs: Tensor, data_samples: SampleList) -> dict:
         """Forward function for FlowNetS when model training.
 
         Args:
-            batch_inputs (Tensor): The concatenated input images.
-            flow_gt (Tensor): The ground truth of optical flow.
-                Defaults to None.
-            valid (Tensor, optional): The valid mask. Defaults to None.
-            img_metas (Sequence[dict], optional): meta data of image to revert
-                the flow to original ground truth size. Defaults to None.
+            inputs (Tensor): Input images of shape (N, 6, H, W).
```

Comment:
inputs (Tensor): The concatenated input images, shaped in (N, 6, H, W). inputs[:, :3] corresponds to img1, while inputs[:, :3] is img2. These images should be mean centered and std scaled.

Reply:
Please view the raw file.

Reply:
Since the parameter name has been changed from ``batch_inputs`` to ``inputs``, it also should be changed in docstring.
Please kindly double check L36~L39, thank you.

Reply:
Thanks!